client/src/RelatedItems/CharacteristicEntry.jsx

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 function CharacteristicEntry({ feature }) {
   return (
     <tr>
-      <td>{Object.values(feature)[0] === 'true' ? '&#10003;' : Object.values(feature)[0]}</td>
-      <td>{Object.keys(feature)[0]}</td>
-      <td>{Object.values(feature)[1] === 'true' ? '&#10003;' : Object.values(feature)[1]}</td>
+      <td className="comp-td">{Object.values(feature)[0][0] === 'true' ? '&#10003;' : Object.values(feature)[0][0]}</td>
+      <td className="comp-td modal-features">{Object.keys(feature)[0]}</td>
+      <td className="comp-td">{Object.values(feature)[0][1] === 'true' ? '&#10003;' : Object.values(feature)[0][1]}</td>
```

Comment:
consider using more descriptive class names to make keeping track of elements easier when styling